doc/deployment/amazon_web_services/aws-deploy-pachyderm.md

```diff
@@ -0,0 +1,238 @@
+# Deploy Pachyderm on AWS
+
+After you deploy Kubernetes cluster by using `kops` or `eksctl`,
+you can then deploy Pachyderm on top of that cluster.
+
+You need to complete the following steps to deploy Pachyderm:
+
+1. Deploy `pachctl` as described in [Deploy pachctl](../../../getting_started/local_installation.html#pachctl).
+1. Add stateful storage for Pachyderm as described in [Add Stateful Storage](#add-stateful-storage).
+1. Deploy Pachyderm by using an [IAM Role](deploying-with-an-iam-role)
+(recommended) or [Static Credentials](deploying-with-static-credentials).
+
+## Add Stateful Storage
+
+In Amazon, stateful data is stored in S3 buckets that reside on Elastic
+Block Storage (EBS) persistent volumes. Pachyderm requires an object
+store for the PFS metadata. Therefore, you need to have at least one
+S3 bucket for a Pachyderm deployment on AWS.
+
+The S3 bucket name must be globally unique across the whole
+Amazon region. Therefore, add a descriptive prefix to the S3 bucket
+name, such as your username.
+Pachyderm recommends that you assign anywher between 1 GB and 10 GB
```

Comment:
typo: anywhere

Reply:
1GB is not enough. We recommend at least a 10GB volume, but if you expect your cluster to be very long running a scale to thousands of jobs a commits, you may want to go larger. It is fairly easy to upgrade later if you choose too small a volume.

Reply:
Also, add a note/reminder:

Note: This volume is for Pachyderm metadata and not for the actual data files which are stored in S3.